DuckDuckGo/TutorialSettings.swift

```diff
@@ -24,7 +24,9 @@ import Core
 struct TutorialSettings {
     
     private struct Keys {
-        static let hasSeenOnboarding = "com.duckduckgo.tutorials.hasSeenOnboarding"
+        // Set the build number of the last build that didn't force them to appear to force them to appear.
+        static let lastBuildWithoutForcePop = 1
```

Comment:
Is the purpose of this as a kind of versioning tracker so we can increment it when there's a new onboarding we want to show? If so, I like it!

If it is about versioning can we change the name here to something more obvious e.g `onboardingVersion`?

Optional extra: I would suggest storing the last version seen in prefs (rather than the current boolean although the public method exposed can still be a boolean) then we can compare if we have a more recent version to show a user than the last version they had seen. This has the benefit of overwriting rather than writing new entries to prefs. But totally optional!